ansible/config_sonic_basedon_testbed.yml

```diff
@@ -220,6 +220,52 @@
       become: true
     when: deploy is defined and deploy|bool == true
 
+  - block:
+    - name: Init restapi keys
+      set_fact:
+        server_key: ""
+        server_crt: ""
+        root_cert: ""
+        dir_path: ""
+
+    - name: read server key
+      set_fact:
+        server_key: "{{ restapi_certs['server_key'] }}"
+      when: restapi_certs['server_key'] is defined
+
+    - name: read server crt
+      set_fact:
+        server_crt: "{{ restapi_certs['server_crt'] }}"
+      when: restapi_certs['server_crt'] is defined
+
+    - name: read root cert
+      set_fact:
+        root_cert: "{{ restapi_certs['root_cert'] }}"
+      when: restapi_certs['root_cert'] is defined
+
+    - name: read directory path
+      set_fact:
+        dir_path: "{{ restapi_certs['dir_path'] }}"
+      when: restapi_certs['dir_path'] is defined
+
+    - name: Create restapi directory
+      file:
+        path: "{{ dir_path }}"
+        state: directory
+        mode: '0755'
+      become: true
+
+    - name: Generate server cert using openssl.
+      command: openssl req \
+          -x509 \
+          -sha256 \
+          -nodes \
+          -newkey rsa:2048 \
+          -keyout "{{ server_key }}"
+          -subj "/CN=sonic.restapi.test"
+          -out "{{ server_crt }}"
+      become: true
+
```

Comment:
look at the telemetry certs deployment, it has 'deploy is defined and deploy|bool == true', why not have that for restapi certs deployment?

Reply:
also, why is there difference between the telemetry cert and restapi cert?

Reply:
if they are the essentially should be the same, then I think we need to unify them into one deployment yml.

Reply:
I have moved the cert creation logic to one yml file.